src/translation.js

```diff
@@ -378,6 +385,25 @@ export default {
 		'about.projekt.svelte':
 			'Das Projekt wurde mit dem Svelte-Framework entwickelt. Dazu wurde Skeleton-UI mit Tailwind.css für das Styling verwendet.',
 		'about.projekt.tests': 'Für die Tests wurde Playwright verwendet',
-		'about.projekt.moreInformations': 'Alle weiteren Informationen findest du in folgendem Video:'
+		'about.projekt.moreInformations': 'Alle weiteren Informationen findest du in folgendem Video:',
```

Comment:
du hast in den Meldungen sowohl "Sie" als auch "du" verwendet, ist nicht so ganz einheitlich